remote/src/main/java/org/buffer/android/boilerplate/remote/BufferooServiceFactory.kt

```diff
@@ -24,7 +24,7 @@ object BufferooServiceFactory {
 
     private fun makeBufferooService(okHttpClient: OkHttpClient, gson: Gson): BufferooService {
         val retrofit = Retrofit.Builder()
-                .baseUrl("https://joe-birch-dsdb.squarespace.com/s/")
+                .baseUrl("https://newsapi.org/v2/")
```

Comment:
It is recommended to store the `BASE_URL` inside global `gradle.properties` also @adesamp